### PR TITLE
Introduce readonly mode (E2D-277)

### PR DESCRIPTION
**Pull request by kkostalkowicz**

### CODE REVIEW

File: src/AppContext.ts [hunk 1]
```diff
@@ -100,9 +102,9 @@ export function setupAppContext() {
   const AssetOverviewViewRxTx =
     inject(
       withModal<AssetsOverviewActionProps, AssetsOverviewExtraProps>(
-        connect<Loadable<CombinedBalances>, AssetsOverviewExtraProps>(
+        connect<Authorizable<Loadable<CombinedBalances>>, AssetsOverviewExtraProps>(
           AssetOverviewView,
-          loadablifyLight(combinedBalances$)
+          authorizablify(() => loadablifyLight(combinedBalances$))
```
Comment (peculiarity) [inline src/AppContext.ts]: I have no idea how you come on with such words.  IMO it should be `authorize`

File: src/blockchain/calls/calls.ts [hunk 2]
```diff
@@ -62,10 +55,30 @@ function calls([context, account]: [NetworkConfig, string]) {
...
 
+export const readCalls$ = context$.pipe(
+  map(readCalls),
+);
+
 export type Calls$ = typeof calls$;
 
 export type Calls = ObservableItem<Calls$>;
+
+export type ReadCalls$ = typeof readCalls$;
```
Comment (peculiarity) [inline src/blockchain/calls/calls.ts]: I would do it differently.

I will have types `Readable | Writable`.
So everywhere where `calls$` are used , I will have a single argument of this type. `Writable` will extends the readable methods with write ones. On higher level you will decide whether to build `calls$` read only or with write access as well. but the passed pipeline will only one

File: src/instant/instantForm.test.ts [hunk 3]
```diff
@@ -29,6 +36,8 @@ const defParams = {
   dustLimits$: of({ DAI: new BigNumber(0.1), WETH: new BigNumber(0.1) }),
   allowances$: of({ DAI: true, WETH: false }),
   calls$: of(defaultCalls) as Calls$,
+  readCalls$: of(defaultReadCalls) as ReadCalls$,
```
Comment (peculiarity) [inline src/instant/instantForm.test.ts]: Continuing the comment above..
here we can have `calls$ : Readable | Writable`

File: src/blockchain/calls/calls.ts (hunk 2 above)
Comment (kkostalkowicz) [reply src/blockchain/calls/calls.ts]: This is also what Maciek suggested, we can talk about it.

**Pull request merged**